docs/IntegrationGuide.md

```diff
@@ -0,0 +1,117 @@
+# Integration guide
+
+## Terminology
+
+- **checkout-order**: an order which customers create during the checkout. It may contain one or more subscriptions.
+- **template-order**: an order which manages a single subscription and is used as a template to create a
+  subscription-order.
+
+## Before you begin
+
+Create all the required custom types and states located in [resources folder](./resources).
+
+## Step 1: Create checkout order with subscriptions
+
+`commercetools-subscriptions` processes only orders with subscription line items.
+
+<details>
+<summary> Click to expand an example order draft. In
+the example, there is 1 subscription line item for wine and 1 line item for pants</summary>
+
+```
+{
+  "orderNumber": "10000",
+  "customerEmail": "test@test.com",
+  "totalPrice": {
+    "currencyCode": "EUR",
+    "centAmount": 18000
+  },
+  "lineItems": [
+    {
+      "name": {
+        "en": "Wine subscription"
+      },
+      "variant": {
+        "sku": "wine01"
+      },
+      "price": {
+        "value": {
+          "type": "centPrecision",
+          "currencyCode": "EUR",
+          "centAmount": 18000,
+          "fractionDigits": 2
+        },
+        "country": "DE"
+      },
+      "quantity": 1,
+      "custom": {
+        "type": {
+          "typeId": "type",
+          "key": "checkout-order-line-item"
+        },
+        "fields": {
+          "cutoffDays": 5,
+          "reminderDays": 5,
+          "subscriptionKey": "",
+          "isSubscription": true,
+          "schedule": "0 0 1 Feb,May,Aug,Nov *"
+        }
+      }
+    },
+    {
+      "name": {
+        "en": "Pants"
+      },
+      "variant": {
+        "sku": "pants"
+      },
+      "price": {
+        "value": {
+          "type": "centPrecision",
+          "currencyCode": "EUR",
+          "centAmount": 8000,
+          "fractionDigits": 2
+        },
+        "country": "DE"
+      },
+      "quantity": 1
+    }
+  ],
+  "custom": {
+    "type": {
+      "typeId": "type",
+      "key": "checkout-order"
+    },
+    "fields": {
+      "hasSubscription": true
+    }
+  }
+}
+```
+
+</details>
+
+Checkout order with subscriptions contains following custom fields:
+
+### Checkout order custom fields:
```

Comment:
```suggestion
### checkout-order custom fields:
```

Reply:
I would use `checkout order`. `checkout` is an adjective describing the noun `order`, same as `template order` or `big order`.  No need to put `-` in between.